app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainViewModel.kt

```diff
@@ -32,54 +32,54 @@ class MainViewModel : ViewModel() {
-            var gCasesPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("total_cases_per_1m_population")
-            var gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
-            var gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
+            val gCases: ArrayList<String>? = gdp.getGlobalCovidData("total_cases")
+            val gActive: ArrayList<String>? = gdp.getGlobalCovidData("active_cases")
+            val gTotalDeath: ArrayList<String>? = gdp.getGlobalCovidData("total_deaths")
+            val gRecovered: ArrayList<String>? = gdp.getGlobalCovidData("total_recovered")
+            val gNewCases: ArrayList<String>? = gdp.getGlobalCovidData("new_cases")
+            val gNewDeaths: ArrayList<String>? = gdp.getGlobalCovidData("new_deaths")
+            val gSeriousCritical: ArrayList<String>? = gdp.getGlobalCovidData("serious_critical")
+            val gCasesPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("total_cases_per_1m_population")
+            val gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
+            val gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
             delay(2000)
 
 
             /**
              * Set fetched data to DTO objects
              */
-            var globalDataDto = GlobalData(gCases!!.get(0), gActive!!.get(0),
+            val globalDataDto = GlobalData(gCases!!.get(0), gActive!!.get(0),
                 gTotalDeath!!.get(0), gRecovered!!.get(0), gNewCases!!.get(0),
                 gNewDeaths!!.get(0), gSeriousCritical!!.get(0), gCasesPerMillion!!.get(0),
                 gDeathPerMillion!!.get(0), gStatistics!!.get(0))
 
             gmData.value = globalDataDto
 
-            var cbc : CasesByCountryDataProvider = CasesByCountryDataProvider()
+            val cbc : CasesByCountryDataProvider = CasesByCountryDataProvider()
 
             /**
              * Call API to retrieve cases by country data
              */
-            var country: ArrayList<String>? = cbc.getCasesByCountryData("country_name")
-            var cases: ArrayList<String>? = cbc.getCasesByCountryData("cases")
-            var deaths: ArrayList<String>? = cbc.getCasesByCountryData("deaths")
-            var active: ArrayList<String>? = cbc.getCasesByCountryData("active_cases")
-            var totalRecovered: ArrayList<String>? = cbc.getCasesByCountryData("total_recovered")
-            var newDeaths: ArrayList<String>? = cbc.getCasesByCountryData("new_deaths")
-            var newCases: ArrayList<String>? = cbc.getCasesByCountryData("new_cases")
-            var seriousCritical: ArrayList<String>? = cbc.getCasesByCountryData("serious_critical")
-            var totalCasesPerMillionPopulation: ArrayList<String>? = cbc.getCasesByCountryData("total_cases_per_1m_population")
```

Comment:
Changed as the variables remained constant so the variable would be better off as a val